### PR TITLE
Fix Server error

### PR DESCRIPTION
**Pull request by Joerg-Schoemer**
* check if d1 is not nil before call getTime

### CODE REVIEW

**Comment by sulkaharo** [comment]
Under which conditions are you seeing this trigger a null pointer exception?

**Comment by Joerg-Schoemer** [comment]
I'm not sure which of the following requests trigger the null-pointer:
/api/v1/treatments.json?token=<token>&find[eventType]=Sensor%20Change&count=1
/api/v1/treatments.json?token=<token>&find[eventType]=Site%20Change&count=1
maybe both!

File: lib/api/treatments/index.js [hunk 1]
```diff
@@ -63,18 +63,20 @@ function configure (app, wares, ctx, env) {
       }
     });
 
-    if (!_isNil(d1)) res.setHeader('Last-Modified', d1.toUTCString());
```
Comment (Joerg-Schoemer) [inline lib/api/treatments/index.js:66]: it is possible that d1 is null, but the next call d1.getTime() was not 'not null safe'.

**Comment by Joerg-Schoemer** [comment]
I‘ve  found the🪳
When no treatment is found by the query, d1 is null.

**Pull request merged**